guides/common/modules/proc_configuring-smart-proxy-for-host-registration-and-provisioning.adoc

```diff
@@ -0,0 +1,57 @@
+[id="configuring-{smart-proxy-context}-for-host-registration-and-provisioning_{context}"]
+= Configuring {SmartProxy} for Host Registration and Provisioning
+
+Use this procedure to configure {SmartProxy} so that you can register and provision hosts using your {SmartProxyServer} instead of your {ProjectServer}.
+
+.Procedure
+. Enable the Registration and Templates features on your {SmartProxyServer} and set the registration and template URLs:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {installer-scenario-smartproxy} \
+--foreman-proxy-registration true \
+--foreman-proxy-registration-url "https://_{smartproxy-example-com}_:{smartproxy_port}" \
+--foreman-proxy-templates true \
+--foreman-proxy-template-url "https://_{smartproxy-example-com}_:8000"
+----
+. On your {SmartProxyServer}, open the corresponding ports:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# firewall-cmd --permanent --zone=public --add-port=8000/tcp
+# firewall-cmd --permanent --zone=public --add-port={smartproxy_port}/tcp
+# firewall-cmd --reload
```

Comment:
We already have instructions for these ports in `guides/common/modules/proc_enabling-connections-to-capsule.adoc`. If they followed that then `{smartproxy_port}` should already be open. Port 8000 is more complicated, where it's not opened by default for vanilla Foreman (since there we don't enabled the templates feature by default).